src/components/forms/PlanForm/index.tsx

```diff
@@ -1145,6 +1173,26 @@ const PlanForm = (props: PlanFormProps) => {
                 </div>
               )}
             />
+            {aboutToRetirePlan && formPayLoad && (
+              <Modal
```

Comment:
I think that the modal could be a but less wide - to the point where it just fits the form with like 5% padding.

Reply:
Addressed [here](https://github.com/onaio/reveal-frontend/pull/1493/commits/2338b2786b4b020c3bb734184a2b09502530a881)